WalletWasabi.Fluent/ApplicationStateManager.cs

```diff
@@ -93,11 +94,6 @@ internal ApplicationStateManager(IClassicDesktopStyleApplicationLifetime lifetim
 		_lifetime.ShutdownRequested += LifetimeOnShutdownRequested;
 
 		_stateMachine.Start();
-
-		if (!startInBg)
-		{
-			_stateMachine.Fire(Trigger.Loaded);
```

Comment:
`Trigger.Loaded` is no longer used anywhere. It can be removed from the enum and from the Closed state.